tutorials/object_spark_1_intro.ipynb

```diff
@@ -0,0 +1,680 @@
+    "\n",
+    "**Note** JP writing: I couldn't find the counterpart of `{band}_modelfit_CModel_flux` in dpdd files. So just for the sake of the exercise, I used `magerr_{band}_cModel`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Subset of columns of interest\n",
+    "cols = \"mag_g_cModel, mag_r_cModel, mag_i_cModel\"\n",
+    "\n",
+    "# SQL - register first the DataFrame\n",
+    "df.createOrReplaceTempView(\"full_tract\")\n",
+    "\n",
+    "# Used to be g_modelfit_CModel_flux\n",
+    "sql_command = \"\"\"\n",
+    "    SELECT {}\n",
+    "    FROM full_tract \n",
+    "    WHERE \n",
+    "        tract == 4849 AND\n",
+    "        clean AND\n",
+    "        magerr_g_cModel>0 AND\n",
+    "        magerr_g_cModel<1e16 AND\n",
+    "        magerr_r_cModel>0 AND\n",
+    "        magerr_r_cModel<1e16 AND\n",
+    "        magerr_i_cModel>0 AND\n",
+    "        magerr_i_cModel<1e16\n",
+    "\"\"\".format(cols)\n",
+    "\n",
+    "# Execute the expression - return a DataFrame\n",
+    "df_sub = spark.sql(sql_command)\n",
+    "\n",
+    "print(df_sub.count())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "So now, we have selected what we think are stars. Let's take a look at the colors of these objects"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mag_g_cModel, mag_r_cModel, mag_i_cModel = np.transpose(df_sub.collect())\n",
+    "plt.hist2d(mag_g_cModel - mag_r_cModel,\n",
+    "           mag_r_cModel - mag_i_cModel, \n",
+    "           bins=100,range=[(-1,2),(-1,2)]);\n",
+    "plt.xlabel('$g-r$')\n",
+    "plt.ylabel('$r-i$')\n",
+    "plt.colorbar(label='Number of objects');"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For more information about the stellar locus from models and in Run1.1p see the [DC2 Coadd Run1.1p direct access -- color-color stellar locus](DC2%20Coadd%20Run1.1p%20direct%20access%20--%20color-color%20stellar%20locus.ipynb) tutorial\n",
+    "\n",
+    "Q: What else can you do to improve the star selection?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "__If you want to see more go to [Part II](DC2%20Coadd%20Run1.1p%20GCR%20tutorial%20--%20Part%20II%3A%20Lensing%20Cuts.ipynb)__"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "desc-spark (2.3.2)",
+   "language": "python",
+   "name": "desc-spark"
```

Comment:
when I followed the instruction of spark kernel installation, the kernel installed is called `desc-pyspark`. So this name doesn't match.

Reply:
OK thanks I will correct both. 

For the kernel name, this raises up a bigger question: Can we deliver a standard kernel for DESC member (as is done for desc-python or desc-stack)? For the moment I am implicitly assuming that people will generate it.

Reply:
I think we should provide `desc-pystack` as a standard desc kernel (cc @heather999 on this).

Note that if one follows the instruction in https://github.com/astrolabsoftware/spark-kernel-nersc, the kernel name will be `desc-pystack`, and hence, regardless whether we provide this kernel as a standard desc kernel, it seems to make sense to use `desc-pystack` in this tutorial notebook.

Reply:
Yes I agree (though I think you meant `desc-pyspark` for the name). I would be happy to work on providing and delivering such a kernel for the collaboration. Let me open an issue for discussing this.

Reply:
right, sorry for the typo (fixed)

Reply:
Following #33 , I reran the notebook with the `desc-python-pyspark` kernel.
Information has been updated inside the notebook as well.